### DISCUSSION TITLE
some styles not applied on production side

### DISCUSSION THREAD

**Discussion by fredbouin** [statamic/cms#4534] [Troubleshooting] [answered]
It looks like there are classes that I created in tailwind.css where the styles don't get applied on the production side, everything looks good in dev.

It also looks like if I create new styles in tailwind.css they also don't work once pushed on the production side.

They are all styles used with the classify addon, i can't tell if that part is coincidental or not.

**Comment by edalzell** [comment, marked as answer]
sounds like a browser cache issue, not really anything to do with Statamic.

Are you using the `mix` helpers/tags?

**Reply by fredbouin** [reply]
I am using the mix helper tags when I link my css and js:

        <link rel="stylesheet" href="{{ mix src='css/tailwind.css' }}">
        <script src="{{ mix src='/js/site.js' }}"></script>

I've cleared cache with ```php artisan config:cache``` and ```php please stache:clear``` and ```php please stache:refresh```  cleared browsing data, ​tried multiple browsers, and multiple devices, downloaded a fresh install of firefox, its the same across the board.

**Reply by duncanmcclean** [reply]
Sounds like your styles could be getting ['purged'](https://tailwindcss.com/docs/optimizing-for-production#removing-unused-css). Do all the styles work if you run `npm run dev` instead of `npm run production`?

**Reply by fredbouin** [reply]
@duncanmcclean that sounds right, it works with run dev.

I'm thinking I have to add classify.php to my purge list in tailwind config, but I get an error when deploying with run production after modifying tailwind.config.

**Reply by duncanmcclean** [reply]
What error do you get?

**Reply by fredbouin** [reply]
nvm I think it was a missed coma or something, sorry about that. Adding classify to purge seems to have solved the issue, thanks @duncanmcclean !

**Reply by duncanmcclean** [reply]
No worries - hope you have a great rest of your day!
